android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -369,9 +375,31 @@ public void onPauseWithNative() {
         if (BraveVpnUtils.isVpnFeatureSupported(BraveActivity.this)) {
             BraveVpnNativeWorker.getInstance().removeObserver(this);
         }
+
         super.onPauseWithNative();
     }
 
+    @Override
+    public void onUserLeaveHint() {
+        super.onUserLeaveHint();
+        if (isActivityFinishingOrDestroyed()) return;
+        Tab currentTab = getActivityTab();
+        if (currentTab != null
+                && currentTab.getUrl() != null
+                && isYTVideoUrl(currentTab.getUrl())
+                && !isInPictureInPictureMode()
+                && BackgroundVideoPlaybackTabHelper.isPlayingMedia(currentTab.getWebContents())) {
+            BackgroundVideoPlaybackTabHelper.sendOrientationChangeEvent(
+                    currentTab.getWebContents(), true);
```

Comment:
why do we send `true` as it's in full screen here? Could it be in portrait mode on that stage?

Reply:
does it mean we always enter pip when Activity goes into bg? What about bg playback in that case? Do we show it as pip or full screen?

Reply:
> why do we send `true` as it's in full screen here? Could it be in portrait mode on that stage?

yes, it is in portrait mode on that stage and when we start pip mode, it will show whole activity in pip mode instead of only video in pip. so for that we have to make it full screen before starting pip.

Reply:
> does it mean we always enter pip when Activity goes into bg? What about bg playback in that case? Do we show it as pip or full screen?

not always but most cases we enter pip when activity goes into bg. so we have show it as pip with playback from pip

Reply:
could you make a video on how does it look like when YT is playing and a user hits a home button(backgrounded Brave). I just wonder won't users who just loved to listen YT music in the bg complain that now we show pip video.

Reply:
sure, I'll make a video. 
Yes, may be some user's complain about that. We can add settings to toggle pip for minimized youtube player (portrait mode). wdyt? @timchilds @deeppandya

Reply:
we basically need an option to remove the PiP or to not bring it at all.

Reply:
we discussed that in DMs and there is a way to close the PiP screen and the audio still plays in the bg. I think we are good with it!